components/inputs/docs/input-checkbox.md

```diff
@@ -135,3 +127,9 @@ As an alternative to using the `<d2l-input-checkbox>` custom element, you can st
 </script>
 <d2l-my-checkbox-elem></d2l-my-checkbox-elem>
 ```
+
+## Accessibility
+
+The `d2l-input-checkbox` component follows W3C's best practice recommendations for a [checkbox](https://www.w3.org/WAI/ARIA/apg/patterns/checkbox/). This means that the component works in the following way:
+- The `Space` key is used to select a focused checkbox instead of the `Enter` key
```

Comment:
I think "instead" is the wrong word here, right? Maybe we can rephrase to "... to select a focused checkbox (not the `Enter` key)".